Dockerfile

```
@@ -7,6 +7,8 @@ RUN go build -o changelog .
 
 FROM alpine:latest
 
+RUN apk add git openssh
```

What about adding the `--no-cache` flag to avoid keeping up a cache?

```suggestion
RUN apk add --no-cache git openssh
```